OpenChange/RTFHandler.m

```diff
@@ -1024,17 +1430,31 @@ - (NSMutableData *) parse
         }
       else
         {
-          // We avoid appending NULL bytes
-          if (*_bytes)
-            [_html appendBytes: _bytes  length: 1];
+          c = *_bytes;
+          // We avoid appending NULL bytes or endlines
+          if (c && (c != '\n')) 
+            {
+              if (c < 0x7f) 
```

Comment:
We could have this as a define as well to avoid magic number with a name like `MAX_ASCII_CHAR`?